cmd/decompress/main.go

```diff
@@ -0,0 +1,59 @@
+// decompress is a tool for decompressing compressed files
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/ulikunitz/xz/lzma"
+)
+
+var (
+	usage = func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage: decompress [file]\n")
+		flag.PrintDefaults()
+		os.Exit(1)
+	}
+)
+
+func decompress(infile *os.File, outfile *os.File) error {
```

Comment:
for readability, maybe write it like this:
```suggestion
func decompress(infile io.Reader, outfile io.Writer) error {
```